src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/Editors/ImplicitUsingsEditor/CSharpImplicitUsingsControl.cs

```diff
@@ -55,7 +55,7 @@ internal sealed class CSharpImplicitUsingsControl : Control
             new PropertyMetadata(true, (o, e) => ((CSharpImplicitUsingsControl)o).OnStringListOrSupportedValuesPropertyChanged()));
         
         // Used to suppress event handling during our own updates, breaking infinite loops.
-        private bool updating;
+        private bool _updating;
```

Comment:
Code that was recently moved from CPS uses a different naming format. One of the analyzer updates started issuing warnings for these, so they've been updated as part of this PR.